### PR TITLE
Modified eventhandle to monitor respective mo until deletion.

### PR DESCRIPTION
**Pull request by ragupta-git**
Modified ucseventhandler to support wathching only mo without looking for specific property.
Removes watch_block on deletion of mo.
Also restructured code for more readability.

```
ueh = UcsEventHandle(handle)
ueh.add(managed_object=mo)
```

### CODE REVIEW

File: tests/utils/test_eventhandler.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    print("Inside watch_mo")
+    event_flag = True
+
+
+def test_watch_mo():
+    mo = handle.query_dn(dn)
+    if mo is None:
+        raise ValueError("Mo does no exist")
+    wb_mo = ueh.add(managed_object=mo, call_back=watch_mo)
+    time.sleep(10)
```
Comment (vvb) [inline tests/utils/test_eventhandler.py]: @ragupta-git Can this be lower? 10 seconds seems a lot.

File: tests/utils/test_eventhandler.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    mo = handle.query_dn(dn)
```
Comment (vvb) [inline tests/utils/test_eventhandler.py]: we already have the mo already, this query is probably not required.

File: tests/utils/test_eventhandler.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    assert event_flag
+
+
+def test_watch_mo_remove():
+    mo = handle.query_dn(dn)
+    if mo is None:
+        raise ValueError("Mo does no exist")
+    wb_mo_remove = ueh.add(managed_object=mo)
+    time.sleep(10)
```
Comment (vvb) [inline tests/utils/test_eventhandler.py]: again, 10 seconds seems like a lot

File: tests/utils/test_eventhandler.py (hunk 1 above)
Comment (ragupta-git) [inline tests/utils/test_eventhandler.py]: Changed to 2 seconds.

File: tests/utils/test_eventhandler.py (hunk 2 above)
Comment (ragupta-git) [inline tests/utils/test_eventhandler.py]: removed line 60 `mo = handle.query_dn(dn)`

File: tests/utils/test_eventhandler.py (hunk 3 above)
Comment (ragupta-git) [inline tests/utils/test_eventhandler.py]: Changed to 2 seconds.

**Pull request merged**